docs/feature-hrr-integration.md

```
@@ -0,0 +1,229 @@
+| H₁ holds, H₂ refuted (migration is bigger than 100 LoC) | Ship persistence default-ON in bundled format; defer mmap until split-format follow-up. | adopt + split |
+| H₃ refuted (no disk-cost concern) | Drop the opt-out flag from the design; persistence is unconditional. | adopt + simplify |
+
+---
+
+## Design
+
+### Persistence
+
+**Format.** Per-store directory `<store_root>/.aelfrice/hrr/` containing:
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)